Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12.16.0
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm ci --only=production
+
+COPY . .
+
+ENV PORT=${PORT:-3000}
+USER 1000:1000
+
+CMD ./node_modules/probot/bin/probot.js run --port $PORT ./index.js
```

Comment:
Would it be better to just leave `ENTRYPOINT` and do `RUN` to ensure that Dockerfile is valid?
Something like:
```
ENTRYPOINT ["./node_modules/probot/bin/probot.js"]

# ensure that Dockerfile is valid
RUN ["./node_modules/probot/bin/probot.js", "--version"]
```
Not sure that `probot.js` has --version command, it is just an idea.

Reply:
I think it might not work on Heroku using `$PORT` environment variable.
To variable expansion works properly I need to run using `shell` form, and not `exec`.

Reply:
@diegomarangoni Looks good and I've verified everything works. I recommend adding a `dockerignore` to avoid accidental inclusion of secrets in the image by adding `.env` and `*.pem` to it. Once you add that and update your branch, we are good to merge.

Additionally if you want to, you can `dockerignore` files not needed like docs and images to ensure optimal image size. Though I'm totally fine with image size optimization being a separate PR.